browser/pages/confirm/confirm.html

```diff
@@ -355,6 +355,16 @@
       <span>Red Hat Container Development Kit requires Oracle VirtualBox {{checkboxModel.virtualbox.minimumVersion}} or newer 5.0.x release. Please uninstall detected Oracle VirtualBox and restart Installer to continue.</span>
     </div>
 
+    <div class="message-container error-message" ng-show="virtualization === false">
+      <span class="pficon pficon-error-circle-o"></span>
+      <span>Please enable hardware virtualization for your platform to improve performance.</span>
+    </div>
+
+    <div class="message-container error-message" ng-show="virtualization === undefined">
+      <span class="pficon pficon-error-circle-o"></span>
+      <span>Make sure hardware virtualization is enabled on your platform if supported to improve performance.</span>
```

Comment:
I have not pushed it yet, because I have to test it first, This PR is based on  https://www.virtualbox.org/manual/ch01.html#features-overview stating:

>No hardware virtualization required. For many scenarios, VirtualBox does not require the processor features built into newer hardware like Intel VT-x or AMD-V. As opposed to many other virtualization solutions, you can therefore use VirtualBox even on older hardware where these features are not present

We could change the message to point out it might not work in some cases.